docs/api/model_selection/importances.rst

```diff
@@ -126,11 +134,28 @@ In both cases, because the coefficient may be negative (indicating a strong nega
 
 This method may also be used for instances; but generally there are very many instances relative to the number models being compared. Instead a heatmap grid is a better choice to inspect the influence of features on individual instances. Here the grid is constructed such that the x-axis represents individual features, and the y-axis represents individual instances. The color of each cell (an instance, feature pair) represents the magnitude of the product of the instance value with the feature's coefficient for a single model. Visual inspection of this diagnostic may reveal a set of instances for which one feature is more predictive than another; or other types of regions of information in the model itself.
 
+The same functionality above can be achieved with the associated quick method `feature_importances`. This method will build the ``FeatureImportances`` object with the associated arguments, fit it, then (optionally) immediately show it.
```

Comment:
Not sure how important this is, the other documentation has been adding a header here, e.g.

```
Quick Method
------------
```

This has the effect of creating a link in the table of contents for the quick method when you're on this page. Do you want to standardize on this or does it matter?

Reply:
Ah thank you, yes &mdash; definitely standardization seems like the right way to go! I was going off the [quick methods example](http://www.scikit-yb.org/en/develop/api/cluster/silhouette.html) from the updated `SilhouetteVisualizer` docs. I'll update my docs here (and in my other QM PRs) and in Silhouette for consistency's sake.

Reply:
Sounds good, feel free to merge when you're ready!